automation-swiftui/AutomationAssessment/ContentView.swift

```diff
@@ -1,20 +1,13 @@
-//
-//  ContentView.swift
-//  AutomationAssessment
-//
-//  Created by Nicholas Jones - Mobile iPlayer - Erbium on 29/10/2024.
-//
-
 import SwiftUI
 
 struct ContentView: View {
-    @State private var selectedTag: Tag = Tag.initialDefault
+    @State private var selectedTopic: Topic = Topic.initialDefault
```

Comment:
I felt 'tag' was confusing when 'topic' is the word we use on the actual news app and 'politics' and 'technology' are a topic, not a tag semantically I would say.